### PR TITLE
Build and Push to GCR

### PR DESCRIPTION
**Pull request by bsieber-mozilla**
ENV Vars for app set within GCP Run

### CODE REVIEW

**Comment by bsieber-mozilla** [comment]
Reach out for the endpoint; we have a "live" dev environment. 

See comments in #17 for more.

**Comment by bsieber-mozilla** [comment]
This Workflow uses the `dev` github environment to inject/use secrets.



App-based env vars are currently set internal to GCP:
- GCP Secret Manager (SM) is the place where the secrets are stored
- Cloud Run uses GCP SM pulling desired secrets and injecting them in the application.
- Cloud Run currently is not continuously deployed; manual steps to point to the latest container are currently necessary as alternate paths for CD are investigated.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,58 @@
...
+
 # Jira Bugzilla Integration (JBI)
-System to sync bugs and issues
+System to sync Bugzilla bugs to Jira issues.
+
+### Caveats
+- The system accepts webhook events from Bugzilla
+- The events are transformed into Jira issues
+- The system POSTs an update to the see_also field of Bugzilla bugs
+- Bugs' whiteboard tags are used to determine if they should be synchronized or ignored
```
Comment (leplatrem) [inline README.md:11]: I would move this line in the second position

File: README.md [hunk 2]
```diff
@@ -1,12 +1,58 @@
...
+
+## Action Configuration
+The action configuration uses a YAML file per environment, this file stores all the configuration information for jbi data flows. The process to enable a new data flow will be explained below.
```
Comment (leplatrem) [inline README.md:14]:
```suggestion
The system reads the action configuration from a YAML file, one per environment. Each entry controls the synchronization between Bugzilla tickets with Jira issues.
```

File: README.md [hunk 3]
```diff
@@ -1,12 +1,58 @@
...
+
+
+Below is an full example of an action configuration:
```
Comment (leplatrem) [inline README.md:17]:
```suggestion
Below is a full example of an action configuration:
```

File: README.md [hunk 4]
```diff
@@ -1,12 +1,58 @@
...
+    parameters:
+      jira_project_key: EXMPL
+      whiteboard_tag: example
+```
+
+A bit more about the different fields...
+- `action` (optional)
+    - string
+    - default: [src.jbi.whiteboard_actions.default](src/jbi/whiteboard_actions/default.py)
+    - If using a custom action, place the PYTHONPATH to the custom action python module
```
Comment (leplatrem) [inline README.md:32]:
```suggestion
    - The specified Python module must be available in the `PYTHONPATH`
```

File: README.md [hunk 5]
```diff
@@ -1,12 +1,58 @@
...
+- `contact`
+    - list of strings
+    - If an issue arises with the workflow, communication will be established with this `contact`
```
Comment (leplatrem) [inline README.md:35]:
```suggestion
    - If an issue arises with the workflow, communication will be established with these contacts
```

File: README.md [hunk 6]
```diff
@@ -1,12 +1,58 @@
...
+    - string
+    - Please enter a description; for example, team name or project use-case.
+- `enabled` (optional)
+    - bool [true, false]
+    - default: false
+    - If false, matching events will not be synchronized
+- `parameters` (optional)
+    - dict
+    - default: {}
+    - The parameters will be validated to ensure the selected action has all expected parameters
```
Comment (leplatrem) [inline README.md:47]:
```suggestion
    - The parameters will be validated to ensure the selected action accepts the specified values
```

**Review by leplatrem — DISMISSED** [review]
I think the two comments left still make sense ;)

File: README.md [hunk 7]
```diff
@@ -1,12 +1,58 @@
...
+![test badge](https://github.com/mozilla/jira-bugzilla-integration/actions/workflows/test-build.yaml/badge.svg)
+
 # Jira Bugzilla Integration (JBI)
-System to sync bugs and issues
+System to sync Bugzilla bugs to Jira issues.
+
+### Caveats
+- The system accepts webhook events from Bugzilla
+- The events are transformed into Jira issues
+- The system POSTs an update to the see_also field of Bugzilla bugs
```
Comment (bsieber-mozilla) [inline README.md:10]:
```suggestion
- The system sets the `see_also` field of the Bugzilla bug with the URL to the Jira issue
```

File: README.md [hunk 8]
```diff
@@ -1,12 +1,58 @@
...
+
 # Jira Bugzilla Integration (JBI)
-System to sync bugs and issues
+System to sync Bugzilla bugs to Jira issues.
+
+### Caveats
+- The system accepts webhook events from Bugzilla
+- The events are transformed into Jira issues
+- The system sets the `see_also` field of the Bugzilla bug with the URL to the Jira issue
+- Bugs' whiteboard tags are used to determine if they should be synchronized or ignored
```
Comment (bsieber-mozilla) [inline README.md:11]:
```suggestion
- The system accepts webhook events from Bugzilla
- Bugs' whiteboard tags are used to determine if they should be synchronized or ignored
- The events are transformed into Jira issues
- The system sets the `see_also` field of the Bugzilla bug with the URL to the Jira issue
```

**Comment by bsieber-mozilla** [comment]
Issue #19 resolved; github actions now deploying to dev Cloud Run on PRs

**Pull request merged**